### PR TITLE
Bug/reference file download issue

### PR DESCRIPTION
**Pull request by bhupeshappfoster**
- Fixed issue where some of source files were not added inside /source directory in downloaded zip.
- Fixed issue where target content in TM file is left blank when source entry contains nested fields.

### CODE REVIEW

File: src/elements/Order.php [hunk 1]
```diff
@@ -626,12 +626,12 @@ public function getStatusLabel()
         return $this->statusLabel;
     }
 
-    public function hasTmMissAlignments()
+    public function hasTmMissAlignments($ignoreNew = true)
```
Comment (sidedwards) [inline src/elements/Order.php:629]: @bhupeshappfoster, could we rename this to `hasTmMisalignments()`

**Review by sidedwards — CHANGES_REQUESTED** [review]
@shnsumit, could you please review and test these proposed changes? The `blockId` changes in the block Field Translators seem pretty different compared to our existing logic. I think it would be a good idea to test these with complex nested field structures and multiple target sites before merging.

@bhupeshappfoster, fyi

File: src/elements/Order.php (hunk 1 above)
Comment (bhupeshappfoster) [reply src/elements/Order.php]: Yes, it's done

**Comment by shnsumit** [comment]
> @shnsumit, could you please review and test these proposed changes? The `blockId` changes in the block Field Translators seem pretty different compared to our existing logic. I think it would be a good idea to test these with complex nested field structures and multiple target sites before merging.

@sidedwards  This is tested with couple of Infor entries and local complex nested entries and it looks fine. We can do another round of testing on dev server.

File: src/elements/Order.php (hunk 1 above)
Comment (sidedwards) [reply src/elements/Order.php]: @bhupeshappfoster can you also update the `$hasMissAlignment` variables?

Comment (bhupeshappfoster) [reply src/elements/Order.php]: Replaced all `Miss Alignment` occurrence with `Misalignment`

**Pull request merged**